wordpress-mobile/react-native-libraries-publisher
# Add `react-native-gesture-handler` library with version `2.10.2`

Following up https://github.com/wordpress-mobile/react-native-libraries-publisher/pull/24, this PR adds the Gesture handler library. We no longer need the [forked repository](https://github.com/wordpress-mobile/react-native-gesture-handler) as the main reason for having it was that Reanimated was also provided in a forked repository ([reference](https://github.com/wordpress-mobile/react-native-gesture-handler/pull/2/files#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR194)).

## How to test
1. Run the command `npm install`.
2. Run the command `./gradlew react-native-gesture-handler:publishToMavenLocal -exclude-task prepareToPublishToS3`
3. Observe that the library is built and published successfully in the local folder `~/.m2/repository/org/wordpress-mobile/react-native-libraries`.
